### PR TITLE
PWN-4441 - Add help menu item to NewCreatePinFragment

### PR DESCRIPTION
**Pull request by kuznetsov**
### Jira Ticket

https://p2pvalidator.atlassian.net/browse/PWN-4441

### Description of Work

Added menu item. Action is mocked with snackbar.

### CODE REVIEW

File: app/src/main/java/org/p2p/wallet/auth/ui/pin/newcreate/NewCreatePinFragment.kt [hunk 1]
```diff
@@ -44,6 +44,21 @@ class NewCreatePinFragment :
         }
     }
 
+    private fun FragmentNewCreatePinBinding.initToolbar() = with(toolbar) {
```
Comment (gslevinkov) [inline app/src/main/java/org/p2p/wallet/auth/ui/pin/newcreate/NewCreatePinFragment.kt:47]: how bout' making it as an extension of Toolbar?

**Review by gslevinkov — APPROVED** [review]

File: app/src/main/java/org/p2p/wallet/auth/ui/pin/newcreate/NewCreatePinFragment.kt (hunk 1 above)
Comment (kuznetsov) [reply app/src/main/java/org/p2p/wallet/auth/ui/pin/newcreate/NewCreatePinFragment.kt:47]: Like this `private fun UiKitToolbar.initToolbar()`?

Comment (gslevinkov) [reply app/src/main/java/org/p2p/wallet/auth/ui/pin/newcreate/NewCreatePinFragment.kt:47]: yep

Comment (kuznetsov) [reply app/src/main/java/org/p2p/wallet/auth/ui/pin/newcreate/NewCreatePinFragment.kt:47]: Well, I don't mind

File: app/src/main/java/org/p2p/wallet/auth/ui/pin/newcreate/NewCreatePinFragment.kt [hunk 2]
```diff
@@ -44,6 +45,21 @@ class NewCreatePinFragment :
         }
     }
 
+    private fun UiKitToolbar.initToolbar() {
+        setNavigationOnClickListener { popBackStack() }
+        setOnMenuItemClickListener {
+            when (it.itemId) {
+                R.id.helpItem -> {
+                    // TODO PWN-4362 open help web view
```
Comment (kabramovp2p) [inline app/src/main/java/org/p2p/wallet/auth/ui/pin/newcreate/NewCreatePinFragment.kt:53]: chrome tabs please! =) not webview

Comment (kuznetsov) [reply app/src/main/java/org/p2p/wallet/auth/ui/pin/newcreate/NewCreatePinFragment.kt:53]: Whatever 🙃

File: app/src/main/java/org/p2p/wallet/auth/ui/pin/newcreate/NewCreatePinFragment.kt [hunk 3]
```diff
@@ -44,6 +45,21 @@ class NewCreatePinFragment :
         }
     }
 
+    private fun UiKitToolbar.initToolbar() {
+        setNavigationOnClickListener { popBackStack() }
+        setOnMenuItemClickListener {
+            when (it.itemId) {
+                R.id.helpItem -> {
+                    // TODO PWN-4362 open help
```
Comment (1sakov) [inline app/src/main/java/org/p2p/wallet/auth/ui/pin/newcreate/NewCreatePinFragment.kt:53]: Why did you move this to another ticket? 
Is there any issue to open feedback and support here?

File: app/src/main/java/org/p2p/wallet/auth/ui/pin/newcreate/NewCreatePinFragment.kt [hunk 4]
```diff
@@ -44,6 +45,21 @@ class NewCreatePinFragment :
         }
     }
 
+    private fun UiKitToolbar.initToolbar() {
+        setNavigationOnClickListener { popBackStack() }
+        setOnMenuItemClickListener {
+            when (it.itemId) {
```
Comment (1sakov) [inline app/src/main/java/org/p2p/wallet/auth/ui/pin/newcreate/NewCreatePinFragment.kt:51]: looks like we need `if` here

**Review by 1sakov — CHANGES_REQUESTED** [review]

File: app/src/main/java/org/p2p/wallet/auth/ui/pin/newcreate/NewCreatePinFragment.kt (hunk 3 above)
Comment (kuznetsov) [reply app/src/main/java/org/p2p/wallet/auth/ui/pin/newcreate/NewCreatePinFragment.kt:53]: Just changed the message due to @kabramovp2p comment. Ticket is the same

Comment (1sakov) [reply app/src/main/java/org/p2p/wallet/auth/ui/pin/newcreate/NewCreatePinFragment.kt]: @kuznetsov why you are leaving TODO here? 
Why you are not opening "Support and feedback" screen as it said in the ticket?

Comment (gslevinkov) [reply app/src/main/java/org/p2p/wallet/auth/ui/pin/newcreate/NewCreatePinFragment.kt]: @kuznetsov you can put implementation here, I asked Nina about it yesterday.
Open Intercom window on help item click

Comment (kuznetsov) [reply app/src/main/java/org/p2p/wallet/auth/ui/pin/newcreate/NewCreatePinFragment.kt]: Thanks guys, opened Intercom from here

**Review by isakovch — APPROVED** [review]

**Review by 1sakov — APPROVED** [review]

**Review by kabramovp2p — APPROVED** [review]
